index.html

```diff
@@ -3143,8 +3143,9 @@ <h3>Verify Proof</h3>
 [=cryptographic suite/hash=] algorithm on the <var>transformedData</var>.
           </li>
           <li>
-Let |isProofVerified:valid or invalid| be the result of running |suite|'s
-[=cryptographic suite/verify proof=] algorithm on <var>hashData</var> and <var>proof</var>.
+Let <var data-type='"valid" or "invalid"'>isProofVerified</var> be the result of running |suite|'s
```

Comment:
Having read the [newly linked documentation](https://github.com/w3c/respec/wiki/data-type)... I wonder whether either tweak below does the ReSpec job? If not, I think there remains a need for an issue against the ReSpec shorthand, that it needs a way to accept such values which are valid in the longhand HTML.
```suggestion
Let |isProofVerified:'"valid" or "invalid"'| be the result of running |suite|'s
```

Reply:
or perhaps
```suggestion
Let |isProofVerified:"'valid' or 'invalid'"| be the result of running |suite|'s
```

Reply:
(I would have tested these myself, but I wasn't able to quickly locate a ReSpec playground, if such exists.)

Reply:
Neither of those works. I think it's using a regular expression to parse the shorthand, and any unexpected character makes it fail. https://github.com/w3c/respec/issues/4632.